www/include/monitoring/objectDetails/template/hostDetails.ihtml

```diff
@@ -367,10 +371,10 @@
                 </tr>
             </table>
             <br />
-            <table class="table ListTable linkList">
+            <table class="ListTable">
                 <tr class='list_lvl_1'>
                     <td class="FormHeader" style="width: 50%;border-bottom:1px solid #d1d2d4;">
-                        <h4>{t}Host commands and shortcuts{/t}</h4>
+                        <h4>{t}{$m_mon_host_commands_shortcuts}{/t}</h4>
```

Comment:
{t} is not needed anymore cause translation is done before

Reply:
Hopefully I only did this on that one :)